app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= render partial: 'partials/navigation_menu', locals: {visited: 'recipes'} %>
+<main>
+  <h2><%= @recipe.name %></h2>
+  <div>
+    <p><span>Preparation Time:</span> <%= @recipe.preparation_time %></p>
+    <%= render partial: 'switch_button', locals: {public: @recipe.public} %>
+  </div>
```

Comment:
- [ ] Please improve the layout of this page in order to match the wireframe given. Here is the comparison 👇 
**Given wireframe**
![image](https://user-images.githubusercontent.com/90222110/218268986-67006357-aae7-4383-b3a9-6d3f54d08cd9.png)

**Your design**
![image](https://user-images.githubusercontent.com/90222110/218269007-593ad3f9-b827-4fd1-8700-11cb79f933c3.png)

As you can see there are some to be improved, like the alignment of buttons and please work on the table to be structured well as on wireframe 😃 👍